### ISSUE TITLE
Inquiry about Unusually High PSNR Results with RT-X Net on LLVIP Dataset

### ISSUE THREAD

**Issue by mlxgccc** [jhakrraman/rt-xnet#2]
I conducted experiments using RT-X Net on the LLVIP dataset and observed surprisingly high PSNR values (above 30).  
Why?

**Comment by jhakrraman** [comment]
Thank you so much for pointing it out. I noticed that there was a mismatch between the pretrained model provided on the drive and the one that we had in our system. So, I have updated the pretrained_model now. 

Feel Free to reach out if you have any questions.

**Comment by mlxgccc** [comment]
Thank you for the above answer.I started training from scratch. To verify if it was due to my settings, I also tested the PSNR values using the other models you compared, and they were relatively 5-10 points higher. Could it be that there's an issue with the dataset you provided or a problem with the submitted version? The dataset was obtained from the one you provided on Hugging Face.

**Comment by jhakrraman** [comment]
I did not directly test on the other models, as in that case it would definitely give an advantage to my model, as it was already trained on LLVIP. I first trained all the models that I have compared on LLVIP, and then, after the training, I tested them on the test set provided. This was a necessary step, as none of the models were trained on LLVIP, as it is a multi-modal dataset. Most of the previous models used LOV-v1, LOV-v2, SID, and SMID datasets. 

In the test results, EnlightenGan and Retinexformer were the closest, both in terms of qualitative analysis as well as quantitative analysis. So, at the end, based on these results, I reported them in the paper.

**Comment by mlxgccc** [comment]
Thank you very much for your answer.I should be able to solve it.
